exporter/sentinels.go

```diff
@@ -66,6 +66,16 @@ func (e *Exporter) extractSentinelMetrics(ch chan<- prometheus.Metric, c redis.C
 		}
 		masterAddr := masterIp + ":" + masterPort
 
+		masterCkquorum, err := redis.String(doRedisCmd(c, "SENTINEL", "CKQUORUM", masterName))
+		log.Debugf("Sentinel ckquorum status for master %s", masterName)
+		masterCkquorumStatus := 0
+		masterCkquorumMsg := masterCkquorum
+		if err != nil {
+			masterCkquorumStatus = 1
```

Comment:
What's the intention with setting this to zero and one depending on the error? 

From a cursory look t the docs, looks like if we have `ckquorum` then things are ok, so make it `1`?

Reply:
I was planning to use this metric as a boolean value. If it's 0 meaning the ckquorum result is okay (meaning `err` is empty), otherwise (`err` value is not empty), the master is not okay or can't be failover

Reply:
I see. Would it make more sense to set it to "1" if things are up and ok? Just like the "up" metric?

Reply:
I think you're right. Also forgot to lookup other metrics with a similar setup. I will update this now.

Reply:
Updated